shared/helpers/jest/browser/download.mjs

```diff
@@ -1,28 +1,28 @@
 import { join } from 'path'
 
+import { Browser, Cache, detectBrowserPlatform, install, resolveBuildId } from '@puppeteer/browsers'
 import { paths } from 'govuk-frontend-config'
-import puppeteer from 'puppeteer'
 
 /**
  * Puppeteer browser downloader
  */
 export async function download () {
-  const fetcher = puppeteer.createBrowserFetcher({
-    path: join(paths.root, '.cache', 'puppeteer/chrome')
-  })
+  const browser = Browser.CHROME
+  const platform = detectBrowserPlatform()
+
+  // Download management
+  const cacheDir = join(paths.root, '.cache', 'puppeteer')
+  const cache = new Cache(cacheDir)
 
   // Downloaded versions
-  // @ts-expect-error 'defaultBrowserRevision' is marked @internal
-  const latest = puppeteer.defaultBrowserRevision
-  const versions = fetcher.localRevisions()
+  const buildId = await resolveBuildId(browser, platform, 'stable')
+  const installed = cache.getInstalledBrowsers()
 
   // Download latest browser (unless cached)
-  if (!versions.includes(latest)) {
-    await fetcher.download(latest)
+  if (!installed.some((install) => install.buildId === buildId)) {
+    await cache.clear()
 
-    // Remove outdated browser versions
-    for (const version of versions) {
-      await fetcher.remove(version)
-    }
+    // Install into cache directory
+    await install({ browser, buildId, cacheDir })
```

Comment:
Ouch, that's becoming much more involved to download the browsers 😬 Thanks for digging how to use the new API 😊 

**question** Random thought (which you've likely explored already anyway): any chance we could leave that complexity back in the hands of Puppeteer [by calling their CLI](https://pptr.dev/browsers-api/#cli) for installing Chrome?

Reply:
@romaricpascal Did some more testing and we're good to go

Percy is using the new Puppeteer v20 browser already (Chrome for Testing) via [@percy/puppeteer](https://www.npmjs.com/package/@percy/puppeteer) for screenshots, and I've followed the steps in [**Skipping the asset discovery browser download**](https://docs.percy.io/docs/skipping-asset-discovery-browser-download) to stop it downloading its own

Ready when you are